packages/components/src/2016-12-01/button.js

```diff
@@ -36,17 +36,30 @@ class Button extends HTMLElement {
     `;
     this.shadowEl = shadowRoot.children[0];
 
-    this.state = {};
+    this.state = ButtonState.getInitialState();
+
+    // render initial state
+    this._queueRender();
 
     this.addEventListener('mouseenter', () => {
       if (this.state.disabled) { return; }
       const nextState = ButtonState.enterHover(this.state);
-      this.dispatchEvent(new CustomEvent('change', { detail: nextState }));
+      this.dispatchEvent(new CustomEvent('change', {
+        detail: {
+          type: 'mouseenter',
+          state: nextState,
+        },
```

Comment:
Sorry I mentioned it in the commit notes. If we have multiple events triggering onChange we may want to disambiguate. 

```typescript
import {Select} from '@orion/react';

handleChange(event) {
  if (event.detail.type === 'selectedIndexChange') {
    submitForm();   
  }

  this.setState({ selectState: event.detail.state });
}

render() {
  return <Select options={this.props.options} {...this.state.selectState} onChange={this.handleChange} />;
}
```